core/client/src/main/java/alluxio/client/file/UnderFileSystemFileOutStream.java

```diff
@@ -167,6 +167,7 @@ private void writeToWorker(byte[] b, int off, int len) throws IOException {
    */
   private ByteBuffer allocateBuffer() {
     Configuration conf = ClientContext.getConf();
-    return ByteBuffer.allocate((int) conf.getBytes(Constants.USER_FILE_BUFFER_BYTES));
+    return ByteBuffer
```

Comment:
this line break looks strange, maybe move `ByteBuffer` down or break after `.allocate(`?